### DISCUSSION TITLE
server unavailable (Reloading server)

### DISCUSSION THREAD

**Discussion by mythshuazi** [nuxt/nuxt#16333] [Questions]
I followed Nuxt3 guide to intall a new Nuxt project, the browser open but the application dont work, i got an 503 error (server unavailable ), i didn’t found any similar error ,browser show like the pic, but the code  run well  on my colleague computer: 

![企业微信截图_16690007313355](https://user-images.githubusercontent.com/7909545/202955989-b5f58d67-a92c-41de-9181-fdea238570b1.png)

![16690003677440](https://user-images.githubusercontent.com/7909545/202955402-1f2090a3-ec87-4c3a-93c9-e7d835a5b039.png)

node version: 16.13.1
npm version: 8.1.2
os version: windows 10 22H2

**Comment by zbwroom** [comment]
Have you solved this problem

**Comment by Baroshem** [comment]
Hey would you mind opening an issue with proper Stackblitz reproduction? This is rather a bug report than question or discussion :)
